src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -159,15 +177,24 @@ public Variant getVariant(String toggleName, Variant defaultValue) {
         return ofNullable(toggleRepository.getToggle(toggleName));
     }
 
+    /**
+     * Use more().getFeatureToggleNames() instead
+     * @return a list of known toggle names
+     */
+    @Deprecated()
     public List<String> getFeatureToggleNames() {
         return toggleRepository.getFeatureNames();
     }
 
+    /**
+     * Use more().count() instead
+     */
+    @Deprecated
     public void count(final String toggleName, boolean enabled) {
         metricService.count(toggleName, enabled);
     }
 
-    private Map<String, Strategy> buildStrategyMap(@Nullable Strategy[] strategies) {
+    private static Map<String, Strategy> buildStrategyMap(@Nullable Strategy[] strategies) {
```

Comment:
Our of curiosity: what's the benefit of converting this into a private static method?

Reply:
When I added the new constructor for testing, and had the existing constructor forward to the new constructor, the compiler disallowed calling a member function (`buildStrategyMap`) until the object has been constructed. Making the method `static` worked around that, and was harmless since the method doesn't use any member variables anyways.

I noticed that `defaultToggleRepository` on line 48 used the same approach, so I figured it was due to the same reasoning.